### ISSUE TITLE
Gauge IntelliJ not working as expected with a gradle project

### ISSUE THREAD

**Issue by sguptatw** [getgauge/Intellij-Plugin#276]
**Expected behavior**
Should be able to perform IDE related actions on a gradle project

**Actual behavior**
Unable to perform actions like refactor, navigate, run from the IDE with a gradle project.

**Steps to replicate**
* clone [gradle project](https://github.com/manupsunny/gauge-gradle-sample)
* Open it in the IDE.
* Unable to perform the actions.

**Version**
```
Gauge version: 0.8.5

Plugins
-------
html-report (4.0.1)
java (0.6.4)
```
Gauge IDE
```
0.3.4
```

**Comment by dengayevskiy** [comment]
Hi!

Can I expect a fix for this in the nearest future? This bug prevents us from moving to Gauge.

**Comment by sguptatw** [comment]
@dengayevskiy : Yes, this is a bug. However, a work around for now is to rename the highest specification folder to specs. Then IntelliJ features will be available.
